controllers/api-controller.js

```diff
@@ -1,8 +1,8 @@
 import axios from "axios";
 import * as moviesDao from "../dao/movies-dao.js";
-import {countCommentsByMovieID} from "../dao/comments-dao.js";
+import { countCommentsByMovieID } from "../dao/comments-dao.js";
 
-const API_KEY = process.env.API_KEY ||'k_ouq4szjo';
+const API_KEY = process.env.API_KEY || 'k_7lfgcu9r';
```

Comment:
We will have to remove the hard-coded part if we want to show it as part of our resume.